web-ui/src/main/resources/catalog/components/edit/suggestion/partials/list.html

```diff
@@ -3,14 +3,20 @@
        data-gn-slide-toggle="">
     <i class="fa fa-cogs"/>&nbsp;
     <span data-translate="">suggestions</span>&nbsp;
-    <button type="button" class="btn btn-default pull-right btn-xs"
-            data-ng-click="load()">
-      <i data-ng-hide="loading" class="fa fa-refresh"/>
-      <i data-ng-show="loading" class="fa fa-spinner fa-spin"/>
-    </button>
+    
   </div>
-  <div class="panel-body" data-ng-show="suggestions.length !== 0">
-    <div class="list-group">
+  <div class="panel-body">
+    <div class="btn-toolbar" role="toolbar">
+      <button type="button"
+              class="btn btn-default btn-xs"
+              data-ng-click="load()">
+        <i data-ng-hide="loading" class="fa fa-fw fa-refresh"/>
```

Comment:
Be careful, `<i>` is not self-closing and this is invalid html5, although it will render fine. I know it is already written like that in many places...

Could you fix the ones you added/modified in this PR?

Reply:
I will fix them